cmake/CopyUserPresetTemplate.cmake

```diff
@@ -0,0 +1,11 @@
+function(CopyUserPresetTemplate)
+	if(
+		NOT DEFINED Languages AND
+		NOT DEFINED Applications AND
+		NOT EXISTS "${CMAKE_SOURCE_DIR}/CMakePresets.json" AND
+		NOT EXISTS "${CMAKE_SOURCE_DIR}/CMakeUserPresets.json"
+	)
+		file(COPY "${CMAKE_SOURCE_DIR}/cmake/presets/CMakeUserPresets.json" DESTINATION "${CMAKE_SOURCE_DIR}")
```

Comment:
I thought COPY does not exists in the cmake of v2019? or is just the syntax sllightly different? read/write is there since at least v3.2.0

Reply:
https://cmake.org/cmake/help/latest/command/file.html#copy-file

Reply:
man that manual is hard to read